school_teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [ ] [optional] The Teacher initializer extends the parent's initializer and should also accept parameters that are required for the parent class Person. I would suggest considering the below code.
  - `def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end`

Reply:
Thank you for your suggestions. I will consider them if needed.